src/logic/operate.js

```diff
@@ -0,0 +1,23 @@
+// eslint-disable-next-line
+import Big from 'big.js';
+
+export default function operate(numberOne, numberTwo, operation) {
+  const one = Big(numberOne);
+  const two = Big(numberTwo);
+  if (operation === '+') {
+    return one.plus(two).toString();
+  }
+  if (operation === '-') {
+    return one.minus(two).toString();
+  }
+  if (operation === 'x') {
+    return one.times(two).toString();
+  }
+  if (operation === '÷') {
+    return one.div(two).toString();
+  }
```

Comment:
- [ ] When I try to divide a number with zero let us say `8/0`
![zero](https://user-images.githubusercontent.com/44745855/133482563-3ab508dc-88bc-49b5-b563-b7c56ad9c4e8.png)

Normally if you check in your calculator it is supposed to display an error on the screen please work on that